js/refactor.js

```diff
@@ -0,0 +1,29 @@
+$(function(){ 
+
+  $("#stopButton").click(turnRed);
+  // console.log("when i press stop button do this");
+
+  function turnRed() {
+    $(".bulb").css("background-color","black");
```

Comment:
@agreen2017 I would have made that an expression a function and call it every time I change the color of the light.